### DISCUSSION TITLE
[#1040943] Exit editor mode without relaunching the game

### DISCUSSION THREAD

**Discussion by Xd-pro** [Mojang/minecraft-editor#93] [Feedback]
Currently you can't and it makes going in and out of editor mode for testing annoying

**Comment by novam77** [comment]
causes irritation every time you try to do it again. it would be nice to find a loophole in this

**Comment by minecraftchloe** [comment]
We've moved up the priority of a feature we're referring to internally as "Edit/Test" -- stay tuned, but it should ease a bit of the pain in this very clunky workflow!!
